config/routes.rb

```diff
@@ -143,6 +143,7 @@
         resource :other_assets, only: %i[show update]
         resource :restrictions, only: %i[show update]
         resource :policy_disregards, only: %i[show update]
+        resource :check_answers_incomes, only: %i[show update]
```

Comment:
Currently we define all the check your answers resources as `:check_<something>_answer`.

I think it makes things much easier to understand, but how do you feel about following the same pattern here as well?

```suggestion
        resource :check_income_answers, only: %I[show update]
```

Reply:
Yeah, that's a good point, I'm not sure why I decided to do it that way around but agree 'check_income_answers' makes more sense - have updated this.

I think re. Joel's comment about renaming means_summaries, it would also make sense to rename this to 'check_capital_answers' again so it's consistent with the other check answers pages.

I don't _think_ the state machine changes should have any effect on in progress application, as the new state only applies to the new page, so anyone who is currently on any other page in the means journey will stay in the same existing state of 'provider_assessing_means' and when they reach the new page will enter the new state. Unless I have misunderstood something about how the state machine works? 

One potential issue though is that for anyone currently in the means journey who has completed the income section but is still somewhere in the capital section when this change gets released, they will get to the check_capital_answers (means_summaries) page and won't see any of their answers which are to do with income, as all these sections will have been removed.